src/server/routes/index.js

```diff
@@ -0,0 +1,121 @@
+/* @flow */
+import type { $Request, $Response } from 'express'
+import { Paratii } from 'paratii-lib/dist/paratii'
+import { getParatiiConfig } from 'utils/AppUtils'
+
+const fs = require('fs')
+const path = require('path')
+const paratiiConfig = getParatiiConfig(process.env.NODE_ENV)
+const paratii = new Paratii(paratiiConfig)
+
+// const path = require('path')
+
+// import { Paratii } from 'paratii-lib/dist/paratii'
+// import { getParatiiConfig } from 'utils/AppUtils'
+
+// const paratiiConfig = getParatiiConfig(process.env.NODE_ENV)
+// const paratii = new Paratii(paratiiConfig)
+module.exports = async (req: $Request, res: $Response) => {
+  // $FlowFixMe
+  const route = req.route.path
+  let index = fs.readFileSync(
+    path.resolve(__dirname, '../../../', 'build', 'index.html'),
+    'utf8'
+  )
+  let meta = ''
+  let script = ''
+  // script manager
+  switch (route) {
+    case '/embed/:id':
+      script = '<script type="text/javascript" src="/embed/bundle.js"></script>'
+      break
+    default:
+      script = '<script type="text/javascript" src="/bundle.js"></script>'
+      break
+  }
+
+  // route manager
+  switch (route) {
+    case '/embed/:id':
+    case '/play/:id':
+      const { id } = req.params
+      const video = await paratii.core.vids.get(id)
+
+      if (video !== null) {
+        meta = openGraphHead(meta, video)
+        meta = twitterCardHead(meta, video)
+      } else {
+        meta = notFoundVideo(meta)
+      }
+
+      break
+    default:
+      console.log('default')
+      meta = basicHead(meta)
+      script = '<script type="text/javascript" src="/bundle.js"></script>'
+      break
+  }
+
+  // console.log(index)
+
+  index = index.replace('{{meta}}', meta)
```

Comment:
Is this just a string replacement? I would rather us not do this. Can't we use [express-handlebars](https://github.com/ericf/express-handlebars) instead of fudging together our own template engine?

Reply:
ok